>From c4236347f6c807793bd5c8c6a7432c0e9a2e78c2 Mon Sep 17 00:00:00 2001
From: Simo Sorce <ssorce@redhat.com>
Date: Wed, 13 Apr 2011 15:21:32 -0400
Subject: [PATCH 2/3] memberof: fix calculation of replaced members

We were skipping the check on the next value in the added list when a match
was found for the currentr value being checked.
---
 src/ldb_modules/memberof.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/src/ldb_modules/memberof.c b/src/ldb_modules/memberof.c
index 55c52fdcbbfccb7607190e21954a1439dc8fa57e..41ea0b344e951ffc47a4882896671bd9b37e1719 100644
--- a/src/ldb_modules/memberof.c
+++ b/src/ldb_modules/memberof.c
@@ -2727,6 +2727,7 @@ static int mbof_mod_process(struct mbof_mod_ctx *mod_ctx, bool *done)
                         added->dns[j] = added->dns[j+1];
                     }
                     added->num--;
+                    i--;
                 }
             }
         }
-- 
1.7.4.2

